src/js/components/Menu/__tests__/Menu-test.js

```diff
@@ -140,39 +140,36 @@ describe('Menu', () => {
         <Menu
           id="test-menu"
           label="Test"
-          items={[{ label: 'Item 1', onClick }, { label: 'Item 2' }]}
+          items={[{ label: 'Item 1' }, { label: 'Item 2', onClick }]}
         />
       </Grommet>,
     );
     expect(container.firstChild).toMatchSnapshot();
 
-    // pressing down 3x: first opens the drop,
-    // second moves to the first suggestion
-    // third moves to the last suggestion
+    // pressing space: opens drop
+    // tab moves to menu label
+    // second moves to first item
+    // third moves to second item
     fireEvent.keyDown(getByLabelText('Open Menu'), {
-      key: 'Down',
-      keyCode: 40,
-      which: 40,
+      key: 'Space',
+      keyCode: 32,
+      which: 32,
     });
-    fireEvent.keyDown(getByLabelText('Open Menu'), {
-      key: 'Down',
-      keyCode: 40,
-      which: 40,
+    fireEvent.keyDown(getByLabelText('Close Menu'), {
+      key: 'Tab',
+      keyCode: 9,
+      which: 9,
     });
-    fireEvent.keyDown(getByLabelText('Open Menu'), {
-      key: 'Down',
-      keyCode: 40,
-      which: 40,
+    fireEvent.keyDown(getByLabelText('Close Menu'), {
+      key: 'Tab',
+      keyCode: 9,
+      which: 9,
     });
-
-    // moves to the first suggestion
-    fireEvent.keyDown(getByLabelText('Open Menu'), {
-      key: 'Up',
-      keyCode: 38,
-      which: 38,
+    fireEvent.keyDown(getByLabelText('Close Menu'), {
+      key: 'Tab',
+      keyCode: 9,
+      which: 9,
```

Comment:
I think we should add one more fireevent block to close the drop once all items were tabbed.